src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -59,6 +59,16 @@ const terminalConfiguration: IConfigurationNode = {
 			default: 'right',
 			description: localize('terminal.integrated.tabs.location', "Controls the location of the terminal tabs, either to the left or right of the actual terminal(s).")
 		},
+		[TerminalSettingId.DefaultLocation]: {
+			type: 'string',
+			enum: ['editor', 'view'],
```

Comment:
@sbatten should we be referring to this as the terminal view or the terminal panel for the user?

```suggestion
			enum: [TerminalTarget.Editor, TerminalTarget.TerminalView],
```